CHANGELOG.md

```diff
@@ -186,13 +186,14 @@
 - Remove 3 unwanted activities from production
 - Content changes to fields for transaction value and activity identifier
 - Increase the width of the application layout
-- Activity show content is show in tabs for financials and details
```

Comment:
I noticed this was in the wrong place when I rebased. Hoping it's agreeable to fix this as I go on the basis that if another PR beats this change to it, I'll have to address a conflict before this can be merged in.